presto-hudi/src/main/java/com/facebook/presto/hudi/HudiSplitManager.java

```diff
@@ -144,7 +154,9 @@ private ExtendedFileSystem getFileSystem(ConnectorSession session, HudiTableHand
         }
     }
 
-    private Optional<HudiSplit> createHudiSplit(HudiTableHandle table, FileSlice slice, String timestamp, HudiPartition partition)
+    private Optional<HudiSplit> createHudiSplit(
+            HudiTableHandle table, FileSlice slice, String timestamp, HudiPartition partition,
```

Comment:
nit: for long argument list, we tend to put one argument per line.

Reply:
Fixed.